src/main.rs

```diff
@@ -1,8 +1,7 @@
 use std::env;
 use std::fs::File;
 
-fn comm() -> Result<String, String> {
-    let args: Vec<String> = env::args().collect();
+fn comm(args: Vec<String>) -> Result<String, String> {
     if args.len() < 3 {
         return Err("comm needs at least two arguments to compare files!".to_string());
     }
```

Comment:
<i class='graphite__hidden'>[Re: lines 11 to 11]<br/></i>

amazing I can comment on this line :O <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/knjiang/comm-rs/2?utm_source=unchanged-line-comment">Graphite</a>.</span>